packages/components/image/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@beae-ui/image",
+  "version": "1.0.0",
+  "description": "Beae UI | Beae UI | A component to display image in the browser component",
+  "author": "Toan Tran (Albert) <trantoan.fox.97@gmail.com>",
+  "homepage": "https://github.com/beae-labs/beae-ui#readme",
+  "license": "MIT",
+  "sideEffects": false,
+  "main": "src/index.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beae-labs/beae-ui.git",
+    "directory": "packages/components/image"
+  },
+  "bugs": {
+    "url": "https://github.com/beae-labs/beae-ui/issues"
+  },
+  "scripts": {
+    "build": "tsup src --dts",
+    "dev": "pnpm build:fast -- --watch",
+    "clean": "rimraf dist .turbo",
+    "typecheck": "tsc --noEmit",
+    "build:fast": "tsup src",
+    "prepack": "clean-package",
+    "postpack": "clean-package restore"
+  },
+  "dependencies": {
+    "@beae-ui/icon": "workspace:^",
+    "@beae-ui/prop-utils": "workspace:^",
+    "@beae-ui/spinner": "workspace:^",
+    "@beae-ui/styled-system": "workspace:^",
+    "@beae-ui/system": "workspace:^",
+    "@beae-ui/utils": "workspace:^"
```

Comment:
vue package? I seen u used vue package on some file, but I don't see on here, plz add it to package.json